feature-utils/poly-sparql/src/query-templates.js

```diff
@@ -0,0 +1,19 @@
+import { polyNs, rdfNs, rdfsNs, xmlNs } from "./namespaces";
+
+export function sparqleInsertTemplate({ graph, triples }) {
+    return `PREFIX poly: ${polyNs} PREFIX rdf: ${rdfNs} PREFIX rdfs: ${rdfsNs} PREFIX xml: ${xmlNs} INSERT DATA { ${
```

Comment:
This line is constant, but you're generating it once in every call. Maybe define it at the top as a constant?